common/djangoapps/edxmako/paths.py

```diff
@@ -63,7 +62,7 @@ def add_directory(self, directory, prepend=False):
         self._collection.clear()
         self._uri_cache.clear()
 
-    def adjust_uri(self, uri, calling_uri):
+    def adjust_uri(self, uri, relativeto):
```

Comment:
This commit is squashed incorrectly, since it's bringing in irrelevant stuff.

Reply:
Any suggestions ?